utils/spec-styles.js

```diff
@@ -9,6 +9,8 @@ const styleLoader = require('@skyux/theme/utils/node-js/style-loader');
 // for the HTML hidden property defined in sky.scss has been applied.
 (function () {
   beforeAll(function (done) {
-    styleLoader.loadStyles().then(done);
+    styleLoader.loadStyles().then(() => {
```

Comment:
Even though we don't support `jasmine-core@3.99.0`, that version throws an error when we pass the `done` callback as an argument of the `then` function.